eva_submission/retrieve_eload_and_project_from_lts.py

```diff
@@ -0,0 +1,134 @@
+import fileinput
+import logging
+import os
+
+import yaml
+from ebi_eva_common_pyutils import command_utils
+from ebi_eva_common_pyutils.config import cfg
+from ebi_eva_common_pyutils.logger import logging_config as log_cfg
+
+from eva_submission.eload_submission import Eload
+from eva_submission.submission_config import EloadConfig
+
+logger = log_cfg.get_logger(__name__)
+
+
+class ELOADRetrieval(Eload):
+    def __init__(self, eload_number, config_object: EloadConfig = None):
+        super().__init__(eload_number, config_object)
```

Comment:
I didn't consider inherited from `Eload` class. It will create a default config and will be saved in the default location. See the `__init__` [here](https://github.com/EBIvariation/eva-submission/blob/7c9fa0c21d81b5fee7fa77dd5e2bb624d43c0770/eva_submission/eload_submission.py#L37). It will also create all the directories. 
Since we're not using anything from `Eload`, I think that isn't necessary and might end up being confusing.

Reply:
updated - no longer inheriting from Eload.